### DISCUSSION TITLE
Schedule setting error that crosses from pm to am... works with current IC10PS settings on panel?

### DISCUSSION THREAD

**Discussion by samot21** [tagyoureit/nodejs-poolController#847] [General]
Awesome App!!!

Only configuration item i ran into was changing my schedule from always to only when active ... the uniqueness of this one schedule item is that is has been setup to start in evening and finishes the following day.. 11p – 4:55a  … I had this in IntelliCenter schedule but it barks when I went to change to When Active in the Controller software?

Firmware 1.064

Thoughts and better... best practice?

Should I break up this schedule from 11p-4:55a in two or should it work with no issues in njspC?

![ScheduleError](https://github.com/tagyoureit/nodejs-poolController/assets/48191653/81f79c46-15c1-4d9b-bb58-44a89660117d)

**Comment by rstrouse** [comment]
Huh I guess this has changed since 1.47 and it slipped by. You used to not be able to do this.  Should be a straight forward fix since Nixie already supports this.
